tests/test_restrict_useradd.py

```diff
@@ -12,14 +12,17 @@ def test_fail_gid():
     assert "Refusing to use a gid of 0" in result.stdout.decode()
 
 
-uids = [0, 100]
+# Changed restriction to only disallow root due to
+# various win/mac hypervisor issues
+# Leaving as a list in case we need to add more restrictions later
+uids = [0]
```

Comment:
The parametrization should  just be removed. If it's not serving any purpose, it just makes the test confusing, and it's not hard to add back later.